main.go

```diff
@@ -57,6 +57,18 @@ func main() {
 	service.DriverConfigParamsFile = *driverConfigParamsfile
 	service.KubeConfig = *kubeconfig
 
+	// Run the service as a pre-init step.
+	if os.Getenv(gocsi.EnvVarMode) == "MDM-Info" {
```

Comment:
For consistency with other modes ("node" and "controller"), can we make this lower case as well ("mdm-info)".

Reply:
Changed to mdm-info.